hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -219,14 +222,24 @@ static int start_model(OtbnModel &model, unsigned start_addr) {
 // Step once in the model. Returns 1 if the model has finished, 0 if not and -1
 // on failure. If gen_trace is true, pass trace entries to the trace checker.
 // If the model has finished, writes otbn.ERR_BITS to *err_bits.
-static int step_model(OtbnModel &model, bool gen_trace, uint32_t *err_bits) {
+static int step_model(OtbnModel &model, bool edn_rnd_data_valid,
+                      uint32_t edn_rnd_data[8], bool edn_urnd_data_valid,
+                      bool gen_trace, uint32_t *err_bits) {
   assert(err_bits);
 
   ISSWrapper *iss = model.get_wrapper(true);
   if (!iss)
     return -1;
 
   try {
```

Comment:
Huh, we use exceptions? Weird.

The style guide doesn't actually allow them, and I don't know if this is OT-owned code or if something that we pulled in that uses them, but we should probably make a note to clean this up when we can.

(I should probably go ahead and add -fno-exceptions to our config just so this doesn't sneak in later on...)

See: https://google.github.io/styleguide/cppguide.html#Exceptions